sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateWorksetDao.java

```diff
@@ -25,7 +25,7 @@
 public class HibernateWorksetDao implements WorksetStore, HibernatePaginatedDataSource<Workset> {
 
   private static final String[] SEARCH_PROPERTIES = new String[] { "alias", "description" };
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "alias" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)